openQA.spec

```diff
@@ -46,7 +46,7 @@
 %define python_scripts_requires %{nil}
 %endif
 %define assetpack_requires perl(Mojolicious::Plugin::AssetPack) => 1.36, perl(CSS::Minifier::XS) perl(JavaScript::Minifier::XS)
-%define common_requires perl(Config::IniFiles) perl(Cpanel::JSON::XS) perl(Cwd) perl(Data::Dump) perl(Data::Dumper) perl(Digest::MD5) perl(Getopt::Long) perl(Minion) => 9.09, perl(Mojolicious) >= 7.92, perl(Try::Tiny) perl(Regexp::Common)
+%define common_requires perl(Config::IniFiles) perl(Cpanel::JSON::XS) perl(Cwd) perl(Data::Dump) perl(Data::Dumper) perl(Digest::MD5) perl(Getopt::Long) perl(Minion) => 9.09, perl(Mojolicious) >= 7.92, perl(Try::Tiny) perl(Regexp::Common), perl(Storable)
```

Comment:
that comma shouldn't be there, only when necessary for more complex statements as in "my-package => 1.0, next"

Reply:
oh. I saw commas already present in that line, that's why I added it

Reply:
It looks strange that we use the `=>` operator for the Minion version. Shouldn't it be `>=` or is this some strange RPM syntax? (This is of course not related to the change.)

Reply:
I don't know, I guess both might work. Wanna try?